package.json

```diff
@@ -6,11 +6,11 @@
   "scripts": {
     "dev": "node server.js",
     "build": "next build",
-    "start": "NODE_ENV=production node server.js",
     "test": "jest",
-    "prettier-js": "prettier --write '{pages,src}/**/*.{js,jsx}'",
```

Comment:
after removing the script below and these targets there will be no way to run the prettier on some files before committing.

Also, we are removing patterns - does it mean that prettier will be on any files at all? Not sure if it's a big deal or now.

Reply:
Prettier can still be run with `yarn yarn format-staged` as explained in the docs contrib guide or `yarn pretty-quick ...` or even `yarn prettier ...` But again, if you prefer to have these here I can bring them back. We should just document their existence and purpose somewhere, at least in the README file?

Reply:
This discussion is the same as in https://github.com/iterative/dvc.org/pull/671#discussion_r331810661 🙂